.github/workflows/continous-integration.yml

```
@@ -61,18 +61,27 @@ jobs:
     name: Wait for docs tests
     runs-on: ubuntu-latest
     needs: [ changes ]
+    # if: needs.changes.outputs.docs == 'true' || github.event_name == 'push' && (startsWith(github.event.ref, 'refs/tags') || github.ref == 'refs/heads/main')
 
     steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+        
       - name: Wait for doc tests
         uses: fountainhead/action-wait-for-check@4699210ccc66e2a13260803fadbb77085421b891
         id: wait-for-doc-tests
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           checkName: Test Documentation
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
-          owner: ${{ github.event.pull_request.head.repo.owner.login || github.actor }}
           timeoutSeconds: ${{ env.WAIT_TIMEOUT_SECS }}
           intervalSeconds: ${{ env.WAIT_INTERVAL_SECS }}
+          
+      - name: fail the step if the doc tests run could not be found
+        if: ${{ steps.wait-for-doc-tests.conclusion == 'cancelled' || steps.wait-for-doc-tests.conclusion == 'failure' }}
```

```suggestion
      - name: Fail the step if the doc tests run could not be found
        if: ${{ steps.wait-for-doc-tests.conclusion == 'timed_out' }}
```

I guess if the "docs tests" fails, that will stop the CI from passing? not sure if that means this one should fail, as it still did its job correctly (same if the conclusion is cancelled)

Looks like there is a specific outcome for if the wait timed out, in which case "waiting for the tests" failed. What do you think?

---

😆You are right. The wait for docs step will fail if the docs test conclusion is 'failure'. The target conclusion here is 'timed_out'. Totally missed the point yesterday.